src/components/UserProfile/TeamsAndProjects/UserTeamsTable.jsx

```diff
@@ -28,18 +28,27 @@ const UserTeamsTable = props => {
             </div>
           )}
           <div className="row">
-            <Col
-              md={props.edit ? '7' : '12'}
+            {/* <Col
+              md={props.edit ? '10' : '12'}
               style={{
                 backgroundColor: ' #e9ecef',
                 border: '1px solid #ced4da',
                 marginBottom: '10px',
               }}
             >
               <span className="teams-span">Teams</span>
-            </Col>
+            </Col> */}
+            <div className="col"
+              style={{
+                backgroundColor: ' #e9ecef',
+                border: '1px solid #ced4da',
+                marginBottom: '10px',
+              }}
+            >
+              <span className="teams-span">Teams</span>
+            </div>
```

Comment:
```suggestion
            <Col
              md={props.edit ? '10' : '12'}
              style={{
                backgroundColor: ' #e9ecef',
                border: '1px solid #ced4da',
                marginBottom: '10px',
              }}
            >
              <span className="teams-span">Teams</span>
            </Col>
```
Should stay consistent with the rest.

Reply:
Please re-review my newest commit. Thanks.